thirdparty/maven/zimbra-maven/debian/rules

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/make -f
+export DEB_BUILD_OPTIONS=nocheck
+
+%:
+	dh $@
+
+override_dh_auto_clean:
+
+override_dh_auto_configure:
+
+override_dh_strip:
+	dh_strip -pzimbra-maven
+	dh_strip -Nzimbra-maven
```

Comment:
Hmm, what are these lines doing?  Seems like -p and -N are opposites... I'm wondering if this whole override is a waste, or if it's accomplishing something that's just non-obvious to me.